src/bugoutClient/bugoutClient.ts

```diff
@@ -0,0 +1,19 @@
+export interface User {
+    user_id: string;
+    username: string;
+    email: string;
+    normalized_email: string;
+    verified: boolean;
+    created_at: string;
+    updated_at: string;
+}
+export interface Auth  {
+    access_token: string;
+    token_type: string;
+};
+
+export type ApiUrls = {
+    broodURL?: string;
+    spireURL?: string;
+} 
+
```

Comment:
We should also have interfaces for `Journal`, `JournalEntry`, etc.

Reply:
I see that you put them in a different file - best to have a single file with all the interfaces. Easier to understand the codebase. I usually call it something like `data.ts`.